### PR TITLE
alerts

### PR DESCRIPTION
**Pull request by anupkgurung**

### CODE REVIEW

**Comment by maitrakhatri** [comment]
looks perfect to me !

**Review by maitrakhatri — APPROVED** [review]

**Review by Creator-singh — APPROVED** [review]
Good to go 🚀

**Review by Naveenchandar — APPROVED** [review]
Looks Good

File: alert.html [hunk 1]
```diff
@@ -0,0 +1,70 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
```
Comment (smilegupta) [inline alert.html:6]: Can add author name and add page information in meta tag

File: alert.html [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+        </span>
+        <span class="material-icons icon">close</span>
+    </div>
+
+    <!-- alert with icon and close button -->
+    <div class="alert-container">
+        <div class="alert flex-row alert-success">        
+            <span class="flex-row">
+                <span class="material-icons mar-rg-sm">check</span>
+                <span><strong>Success</strong> your action is successful.</span>
```
Comment (smilegupta) [inline alert.html:33]: mar-rg-sm? is it saying something margin-right-small? Can improve on class names

**Review by smilegupta — APPROVED** [review]
LGTM, just added some sanity comments

**Pull request closed without merging**